test/helpers/reset-database.js

```diff
@@ -6,12 +6,13 @@ const vetVisits = require('../mocks/schemes/vet-visits')
 const cs = require('../mocks/schemes/cs')
 const bps = require('../mocks/schemes/bps')
 const fdmr = require('../mocks/schemes/fdmr')
+const sfi23 = require('../mocks/schemes/sfi23')
```

Comment:
I'd put this below manual just to match the order of the schemes

Reply:
order amended, thanks!